src/styles/subsocial.scss

```diff
@@ -1950,6 +1950,13 @@ hr {
   -webkit-transition: none !important;
 }
 
+.DfOnBoardingModal {
```

Comment:
I think you can move this to the local sass file for `DfOnBoardingModal` instead of adding it in `subsocial.scss`

Reply:
Resolved in #3e5b088

Reply:
@teodorus-nathaniel